site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -442,7 +474,7 @@ include::../shared/_premium-edition-blurb-api.adoc[]
 :premium_feature!:
 
 [field]#tenant.usernameConfiguration.unique.numberOfDigits# [type]#[Integer]# [optional]#Optional# [default]#defaults to `5`# [since]#Available since 1.27.0#::
-The maximum number of digits to use when building a unique suffix for a username.  A number will be randomly selected and will be 1 or more digits up to this configured value in length. For example, if this value is `5`, the suffix will be a number between `00001` and `99999`, inclusive and left padded with zeros. The value of this field must be greater than or equal to `3` and less than or equal to `10`.
+The maximum number of digits to use when building a unique suffix for a username. A number will be randomly selected and will be 1 or more digits up to this configured value in length. For example, if this value is `5`, the suffix will be a number between `00001` and `99999`, inclusive and left padded with zeros. The value of this field must be greater than or equal to `3` and less than or equal to `10`.
```

Comment:
This won't necessarily be left padded, I think I have an old comment in the code. Essentially it will be a random number of digits up to the length configured.